src/configs/lang.ts

```diff
@@ -0,0 +1,359 @@
+// import the strings reference i18n file for the specified language
+import { LANGUAGE } from './env';
+// tslint:disable-next-line:no-var-requires
+const reference = require('./strings')[LANGUAGE] || {};
+/**
+ * @param {string} key The unique variable name of the display string
+ * @param {string} fallback The (original) English fallback string if reference doesn't include the display string
+ * @returns {string} The translated display string
+ */
+const translate = (key: string, fallback: string): string =>
+  (reference[key] && reference[key].message) || fallback;
```

Comment:
Helper function to return translated display strings from the loaded reference file, falling back to original English strings if no translated display string is provided in the loaded i18n file.